lib/rmt/cli/main.rb

```diff
@@ -1,36 +1,34 @@
 require 'thor'
 require 'terminal-table'
 
-# rubocop:disable Rails/Output
-
 class RMT::CLI::Main < RMT::CLI::Base
 
   class_option :version, desc: 'Show RMT version', type: :boolean, aliases: '-v', required: false
   class_option :debug, desc: 'Enable debug output', type: :boolean, aliases: '-d', required: false
 
+  desc 'sync', 'Sync database with SUSE Customer Center'
+  def sync
+    RMT::SCC.new(options).sync
```

Comment:
1. That has that same issue with first level commands and `--help` flag.
2. `sync` is not very descriptive (mirroring is also a kind of sync).

Reply:
`scc sync` is also not much more descriptive. And it is as easy to confuse with `mirror` than just `sync` (imo), so I don't really care anymore. I can revert if you prefer that.

@kalabiyau  We need some real users (and some planned time for sitting with them) to give us some better understanding on how clear/confusing it is. Before that, we can move commands around all we want and it won't get much better :expressionless:

Reply:
@thutterer Well, `scc sync` doesn't have that `--help` issue. Otherwise I don't care much about this as well :D

Reply:
Neither does `version --help` work :wink: But it really puzzles me why Thor does not support that (but it supports the same thing with subcommands, e.g. `rmt-cli repos enable --help` does work) ... so I'm wondering if we should hack Thor to support it.

For now, I'd like to leave it as it is and get some user insights before we invest any more time into reshuffling the commands layout.